### PR TITLE
Clean Up dataset and datasetVersion DAO with implementation

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/advancedService/AdvancedServiceImpl.java [hunk 1]
```diff
@@ -905,28 +915,62 @@ public void findHydratedDatasetVersions(
...
+              userInfo,
+              request.getIdsOnly(),
+              false,
+              true,
+              request.getSortKey(),
+              request.getAscending());
       LOGGER.debug(
-          "DatasetVersionPaginationDTO record count : "
-              + datasetVersionPaginationDTO.getTotalRecords());
+          "DatasetVersionPaginationDTO record count : " + commitPaginationDTO.getTotalRecords());
```
Comment (coryverta) [inline backend/src/main/java/ai/verta/modeldb/advancedService/AdvancedServiceImpl.java]:
```suggestion
          "CommitPaginationDTO record count : " + commitPaginationDTO.getTotalRecords());
```

**Review by coryverta — APPROVED** [review]

**Comment by conradoverta** [comment]
@ad-47 is this still an issue?

**Comment by anandjakhaniya** [comment]
> @ad-47 is this still an issue?

this is not an issue, this is dead code. Now instead of this we have Repository and commit Service/DAO

File: backend/src/main/java/ai/verta/modeldb/advancedService/AdvancedServiceImpl.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/advancedService/AdvancedServiceImpl.java]: Fixed

**Review by jkwatson-verta — APPROVED** [review]

**Pull request merged**